cmd/create_data_stream.go

```diff
@@ -126,6 +127,80 @@ func createDataStreamCommandAction(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if answers.Type == "logs" {
+		qs := []*survey.Question{
+			{
+				Name: "inputs",
+				Prompt: &survey.MultiSelect{
+					Message: "Select input types which will be used in this data stream. See https://www.elastic.co/docs/reference/fleet/elastic-agent-inputs-list for description of the inputs",
+					Options: []string{
+						"aws-cloudwatch",
+						"aws-s3",
+						"azure-blob-storage",
+						"azure-eventhub",
+						"cel",
+						"entity-analytics",
+						"etw",
+						"filestream",
+						"gcp-pubsub",
+						"gcs",
+						"http_endpoint",
+						"journald",
+						"netflow",
+						"redis",
+						"tcp",
+						"udp",
+						"winlog",
+					},
+					PageSize: 50,
+					Description: func(value string, index int) string {
+						switch value {
+						case "aws-cloudwatch":
+							return "AWS Cloudwatch"
+						case "aws-s3":
+							return "AWS S3"
+						case "azure-blob-storage":
+							return "Azure Blob Storage"
+						case "azure-eventhub":
+							return "Azure Eventhub"
+						case "cel":
+							return "Common Expression Language (CEL)"
+						case "entity-analytics":
+							return "Entity Analytics"
+						case "etw":
+							return "Event Tracing for Windows (ETW)"
+						case "filestream":
+							return "Filestream"
+						case "gcp-pubsub":
+							return "GCP PubSub"
+						case "gcs":
+							return "Google Cloud Storage (GCS)"
+						case "http_endpoint":
+							return "HTTP Endpoint"
+						case "journald":
+							return "Journald"
+						case "netflow":
+							return "Netflow"
+						case "redis":
+							return "Redis"
+						case "tcp":
+							return "TCP"
+						case "udp":
+							return "UDP"
+						case "winlog":
+							return "WinLogBeat"
+						}
```

Comment:
To ensure that we keep this list updated with the options of the multiselect, we could keep this info in a map, and use the ordered keys as options for the multiselect.
```suggestion
						return inputOptions[value]
```

Reply:
Thanks, it's a good idea to use a map. I've done that, so there's now just one map for the names and descriptions, instead of the duplicated lists.